tests/test_database.py

```diff
@@ -0,0 +1,107 @@
+from os import sys, path
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+
+import os
+
+import shelve
+import unittest
+import sqlite3
+
+from data.data_manager import DataManager
+from model.amity_model import Amity
+from model.office import Office
+from model.living import Living
+from cStringIO import StringIO
+
+class TestDatabase(unittest.TestCase):
+    
+    @classmethod
+    def setUpClass(TestDatabase):
```

Comment:
Check the conventional way of writing a class method, use `cls` in place of _TestDatabase_.